package.json

```diff
@@ -28,16 +29,16 @@
   "author": "Simon Tsvilik",
   "license": "MIT",
   "devDependencies": {
-    "babel": "^6.23.0",
-    "babel-cli": "^6.26.0",
-    "babel-preset-env": "^1.7.0",
-    "babel-register": "^6.26.0",
+    "@babel/cli": "^7.4.3",
+    "@babel/core": "^7.4.3",
+    "@babel/preset-env": "^7.4.3",
+    "@babel/register": "^7.4.0",
     "chai": "^4.2.0",
     "coveralls": "^3.0.2",
-    "eslint": "^5.6.1",
+    "eslint": "^5.16.0",
     "gulp": "^3.9.1",
```

Comment:
I also realized that gulp is not used in this project :/

Reply:
I'll remove it, I'll have further improvements to the dependencies in a separate PR